src/Spectre.Console/Prompts/TextPrompt.cs

```diff
@@ -97,7 +97,15 @@ public sealed class TextPrompt<T> : IPrompt<T>, IHasCulture
     public TextPrompt(string prompt, StringComparer? comparer = null)
     {
         _prompt = prompt ?? throw new System.ArgumentNullException(nameof(prompt));
-        _comparer = comparer;
+
+        if (typeof(T) == typeof(string) && comparer is null)
+        {
+            _comparer = StringComparer.OrdinalIgnoreCase;
+        }
+        else
+        {
+            _comparer = comparer;
+        }
```

Comment:
The comparer is always on the underlying string type, so if we want to default to a specific one, I think it will suffice to make this code look like this:

```suggestion
        _comparer = comparer ?? StringComparer.OrdinalIgnoreCase;
```